### ISSUE TITLE
Offworld oil drills not producing after restart

### ISSUE THREAD

**Issue by jonathan-ng** [GTNewHorizons/GT-New-Horizons-Modpack#6842]
#### Which modpack version are you using?

2.0.9.0 QF4

#
#### If in multiplayer; On which server does this happen? 

Single-player

#
#### What did you try to do, and what did you expect to happen?

Saltwater drilling on the moon. I placed an admin anchor and excess power, and left the moon to do other things.
Expected to return to a nice big super tank of salt water.

#
#### What happened instead? (Attach screenshots if needed)

Very little saltwater found in tank. After a few subsequent attempts, the amount seemed consistent with time from last visit to the moon to end of session. Chunkloading with FTBU did not seem to help.

I made a test world with the drill attached to an ender tank to observe the problem and confirmed that the drill was not producing after a restart. 

I tested a Galacticraft command /gckeeploaded dim_id which immediately solved the problem. 

#
#### What do you suggest instead/what changes do you propose?

According to the Galacticraft wiki, this command shouldn't be necessary for planets. However, I noticed that the command changed the config file \Galacticraft\core.conf, which contains a list of statically loaded dimensions that only had the overworld 0. 

I'm not sure why there is a problem, or why running this command fixes it, but changing the default config file to include the galacticraft planets should be able to work around it.

**Comment by Prometheus0000** [comment]
Keep in mind GC planets are incredibly laggy, so it may have been turned off on purpose. And if other people were having this problem, it would've come up by now. Try removing FTBU and only using the admin anchors and see if that works, maybe it doesn't work right.

**Comment by jonathan-ng** [comment]
I didn't have FTBU when I first had this problem, and installed it to try to fix it, but bafflingly, after uninstalling it and FTBLib, it seems to be working in my test world. I decided to cheat in an ender tank in my survival world, and it seems to work. 

I'm quite certain there was previously a problem. In the brief existence of the cheated endertank, i got 70B of saltwater, yet in the time it takes to research a couple of hundred TC researches, eat a lootbag pure tear and wait out the warp effects, I only got 450B.
